Rakefile

```diff
@@ -1,2 +1,10 @@
 require "bundler/gem_tasks"
 
+begin
+  require "rspec/core/rake_task"
+  RSpec::Core::RakeTask.new(:spec)
+
+  task default: :spec
+rescue LoadError
+end
+
```

Comment:
Can you kill the extra whitespace here?

Reply:
Sure =) 8cea7992236c20f959de994ed32621d69b26e66f